app/views/static_pages/start_page.html.erb

```diff
@@ -0,0 +1,36 @@
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-xl">
+        <%= t('student_loans.journey_name') %>
+      </h1>
+
+      <p class="govuk-body">
+        Claim student loan repayments if you:
+      </p>
+
+      <ul class="govuk-list govuk-list--bullet">
+        <li>
+          teach biology, chemistry, physics, computer science or languages (not including English)
+          for over half your working hours, including your planning, preparation and assessment (PPA) time
+        </li>
+        <li>have been teaching for 1 to 5 years</li>
+        <li>teach in certain places - <a href="https://www.gov.uk/guidance/teachers-student-loan-reimbursement-guidance-for-teachers-and-schools#eligibility-criteria">check which places are eligible</a>.</li>
+        <li>received qualified teacher status in or after the 2013 to 2014 academic year</li>
+      </ul>
+
+      <h2 class="govuk-heading-m">Before you apply</h2>
+
+      <p class="govuk-body">You'll need:</p>
+
+      <ul class="govuk-list govuk-list--bullet">
+        <li>the exact amount you have repaid – get this from your annual student loan statement, your P60 or by contacting the <a href="https://www.gov.uk/guidance/contact-slc-repayment-enquiries">student loans company</a>.</li>
```

Comment:
As above I think we can remove the full stop, a minor thing but I think worth doing. Also I'm aware that I probably put that in on the original Heroku version :)